tests/unit/dataactvalidator/test_b9_award_financial.py

```diff
@@ -44,45 +44,51 @@ def test_success_null(database):
 def test_success_fiscal_year(database):
     """ Testing valid name for FY that matches with budget_year"""
 
-    af = AwardFinancialFactory(row_number=1, submission_id='1', agency_identifier='test',
-                               main_account_code='test', program_activity_name='test',
-                               program_activity_code='test')
+    af_1 = AwardFinancialFactory(row_number=1, submission_id='1', agency_identifier='test',
+                                 main_account_code='test', program_activity_name='test',
+                                 program_activity_code='test')
+
+    af_2 = AwardFinancialFactory(row_number=1, submission_id='1', agency_identifier='test2',
+                                 main_account_code='test2', program_activity_name='test2',
+                                 program_activity_code='test2')
 
     pa_1 = ProgramActivityFactory(budget_year=2016, agency_id='test', allocation_transfer_id='test',
                                   account_number='test', program_activity_name='test', program_activity_code='test')
 
     pa_2 = ProgramActivityFactory(budget_year=2017, agency_id='test2', allocation_transfer_id='test2',
                                   account_number='test2', program_activity_name='test2', program_activity_code='test2')
 
-    submission = SubmissionFactory(submission_id='1', reporting_fiscal_year='2016')
```

Comment:
If they submit on `reporting_fiscal_year 2017` and the PA name doesn't exist in the 2017 PA name, it should look for PA name in 2016 instead. 

So we want to test that PA name (test) for `butget_year=2016` which should pass even for the submission `reporting_fiscal_year 2017`